common/config/rush/pnpm-lock.yaml

```diff
@@ -799,6 +799,7 @@ importers:
       '@aws/eslint-config-workbench-core-eslint-custom': workspace:*
       '@aws/workbench-core-audit': workspace:*
       '@aws/workbench-core-logging': workspace:*
+      '@hapi/boom': ^10.0.0
```

Comment:
Don't we already have `boom` in the project? Is this a major version upgrade?

Reply:
we dont have it in base, only in other packages